File: test/java/test_Penman.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double evapoTranspirationPriestlyTaylor = 449.367;
        double hslope = 0.584;
        double VPDair = 2.19;
        double psychrometricConstant = 0.66;
        double Alpha = 1.5;
        double lambdaV = 2.454;
        double rhoDensityAir = 1.225;
        double specificHeatCapacityAir = 0.00101;
        double conductance = 598.685;

        Penman res0 = Estimation_Penman.CalculatePenman(evapoTranspirationPriestlyTaylor,hslope,VPDair,psychrometricConstant,Alpha,lambdaV,rhoDensityAir,specificHeatCapacityAir,conductance);
        System.out.println(" evapoTranspirationPenman: "+res0.evapoTranspirationPenman);

        System.out.println(((new BigDecimal(res0.evapoTranspirationPenman)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(830.958)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}

File: test/cs/test_Penman.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        Penman res0 = Estimation_Penman.CalculatePenman(specificHeatCapacityAir:0.00101,rhoDensityAir:1.225,VPDair:2.19,conductance:598.685,psychrometricConstant:0.66,evapoTranspirationPriestlyTaylor:449.367,Alpha:1.5,lambdaV:2.454,hslope:0.584);

        Console.WriteLine("{0}","evapoTranspirationPenman: "+res0.evapoTranspirationPenman);

        Console.WriteLine("evapoTranspirationPenman Comparison: ("+Math.Round(830.958,3)+";"+Math.Round(res0.evapoTranspirationPenman,3)+") "+Equals(Math.Round(830.958,3),Math.Round(res0.evapoTranspirationPenman,3)));
    }
}
